addons/resource.language.en_gb/resources/strings.po

```diff
@@ -4163,7 +4163,10 @@ msgctxt "#1019"
 msgid "Enter the username"
 msgstr ""
 
-#empty string with id 1020
+#: xbmc/dialogs/GUIDialogMediaSource.cpp
```

Comment:
Please also add a verbal description for the purpose of this label to help our translators. 

#. …

Reply:
@ksooo Is that the correct way?

Reply:
yes and please remove GUIDialogMediaSource.cpp from message 231

Reply:
> yes and please remove GUIDialogMediaSource.cpp from message 231

Done